README.md

```diff
@@ -32,8 +32,8 @@ Here are the principles we use to guide our development.  From [opensearch.org](
 ## Getting Started
 
 To run OpenSearch Dashboards locally, you first need build artifacts from OpenSearch.
-* Clone the OpenSearch repo with ```git clone git@github.com:opensearch-project/OpenSearch.git```
-* Follow installation and setup instructions in the OpenSearch repo
+* Clone the OpenSearch repo with ```git clone http://github.com/opensearch-project/OpenSearch-Dashboards.git```
```

Comment:
Should be `https`. 

Generally git/https is a matter of preference, but we can use https since it's the default.

Reply:
Should be good now.